src/npcmove.cpp

```diff
@@ -1075,8 +1076,21 @@ bool npc::alt_attack_available()
     for( auto &elem : ALT_ATTACK_ITEMS ) {
         if( ( !is_following() || rules.use_grenades ||
               !( item::find_type( elem )->item_tags.count( "GRENADE" ) ) ) &&
-            has_amount( elem, 1 ) ) {
-            return true;
+                has_amount( elem, 1 ) ) {
+            item *used = nullptr;
+            invslice slice = inv.slice();
+            for (size_t i = 0; i < inv.size(); i++) {
+                if (slice[i]->front().type->id == elem) {
+                    used = &(slice[i]->front());
+                }
+            }
+            const auto use_fun = used->get_use( "iuse_transform" );
+            const iuse_transform *actor = dynamic_cast<const iuse_transform *>( use_fun->get_actor_ptr() );
```

Comment:
This line causes a crash when `use_fun` is null, there needs to be a check for null `use_fun` before this line.